### ISSUE TITLE
WGAN problem

### ISSUE THREAD

**Issue by darinshaw312** [YangNaruto/FQ-GAN#3]
Hi! I can't train my model with your FQ-module based on WGAN, gradient explosion suffered, Can you train FQGAN based on WGAN？thank you very much ^_^

**Comment by YangNaruto** [comment]
Sure. What kind of dataset and codebase you use? Actually, the StyleGAN defines Wasserstein distance   in its loss.py. You may modify it accordingly.

**Comment by darinshaw312** [comment]
I am a GAN beginner in SR, so I try to implement your FQ-part in my discriminator network, behind block module, code framework is based on ProGAN and dataset is CelebA-HQ. I find that training process seems stable from the observation of reconstructed image, when I set the weight of WGAN adversarial loss to 0.05, but I am not sure if it's caused by this operation,  because at the beginning of training, the loss value is norm, and just suddenly collapse when weight of adversarial loss item sets 1. Is the influence of hyperparameter so great?
